### PR TITLE
catch the securesystemslib exception add supply advice

### PR DESCRIPTION
**Pull request by lixuefeng2**
Signed-off-by: lixuefeng (Cloud) <li.xuefeng@h3c.com>

Please fill in the fields below to submit a pull request.  The more information
that is provided, the better.

**Fixes issue #**:

**Description of the changes being introduced by the pull request**:
Sometimes for starter, the securesystemslib may failed to installed correctly. Init the repo shall get hard-to-understand exception message. This modification is to guide user to solve this problem.

### CODE REVIEW

**Comment by lixuefeng2** [comment]
@sebastien , It seems the CI has problems? I don't know why it just only raise exception in python 3.6.
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\projects\\tuf-2aew1\\tests\\tmpv9tr3to2\\TestReplayAttack_x_lr16lz\\repository\\metadata\\timestamp.json'

**Comment by lukpueh** [comment]
The error indeed seems unrelated. @awwad, have you seen this before?

File: tuf/scripts/repo.py [hunk 1]
```diff
@@ -1141,6 +1141,10 @@ def parse_arguments():
   except (tuf.exceptions.Error) as e:
     sys.stderr.write('Error: ' + str(e) + '\n')
     sys.exit(1)
-
+  except (securesystemslib.exceptions.UnsupportedLibraryError) as e:
+    sys.stderr.write('Error: ' + str(e) + '\n')
+    sys.stderr.write('resolve this problem by:\n')
+    sys.stderr.write('sudo pip install securesystemslib[crypto,pynacl]\n')
```
Comment (lukpueh) [inline tuf/scripts/repo.py:1147]: I think it's better to point the user to the [TUF installation instructions](https://github.com/theupdateframework/tuf/blob/develop/docs/INSTALLATION.rst) here.

**Comment by lukpueh** [comment]
Closing due to inactivity. Re-visit with #811.

**Pull request closed without merging**